cd/python/pypi/pypi_publish.py

```diff
@@ -34,11 +34,11 @@ def post_wheel(path):
     """
     logging.info('Posting {} to PyPI'.format(path))
     pypi_credentials = get_secret()
+    os.environ['TWINE_USERNAME'] = pypi_credentials['username']
+    os.environ['TWINE_PASSWORD'] = pypi_credentials['password']
```

Comment:
Can you unset them in a final block? Sometimes we print env vars for debugging purposes in case of errors. Don't want them to be leaked

Reply:
Instead of writing to the environment variable of current process, I moved the credentials to the subprocess environment variable.

Reply:
Even better, great idea!